addons/stock_picking_batch/views/stock_picking_type_views.xml

```diff
@@ -7,30 +7,68 @@
                         </div>
                         <span invisible="not auto_batch"/>
                         <div name="batch_destination" class="o_row" invisible="not auto_batch">
-                            <field name="batch_group_by_destination"/>
+                            <field name="batch_group_by_destination" class="oe_inline"/>
                             <label for="batch_group_by_destination"/>
                         </div>
                         <span invisible="not auto_batch or not default_location_src_id" groups="stock.group_stock_multi_locations"/>
                         <div name="batch_source_location" class="o_row" invisible="not auto_batch or not default_location_src_id" groups="stock.group_stock_multi_locations">
-                            <field name="batch_group_by_src_loc"/>
+                            <field name="batch_group_by_src_loc" class="oe_inline"/>
                             <label for="batch_group_by_src_loc"/>
                         </div>
                         <span invisible="not auto_batch or not default_location_dest_id" groups="stock.group_stock_multi_locations"/>
                         <div name="batch_dest_subloc" class="o_row" invisible="not auto_batch or not default_location_dest_id" groups="stock.group_stock_multi_locations">
-                            <field name="batch_group_by_dest_loc"/>
+                            <field name="batch_group_by_dest_loc" class="oe_inline"/>
                             <label for="batch_group_by_dest_loc"/>
                         </div>
-                        <field name="batch_max_lines" invisible="not auto_batch"/>
-                        <field name="batch_max_pickings" invisible="not auto_batch"/>
+                        <span class="o_form_label fw-bold" invisible="not auto_batch">Wave Grouping</span>
+                        <div name="wave_product" class="o_row" invisible="not auto_batch">
+                            <field name="wave_group_by_product"/>
+                            <label for="wave_group_by_product"/>
+                        </div>
+                        <span invisible="not auto_batch"/>
+                        <div name="wave_category" class="o_row" invisible="not auto_batch">
+                            <field name="wave_group_by_category"/>
+                            <label for="wave_group_by_category"/>
+                        </div>
+                        <span invisible="not auto_batch" groups="stock.group_stock_multi_locations"/>
+                        <div name="wave_location" class="o_row" invisible="not auto_batch" groups="stock.group_stock_multi_locations">
+                            <field name="wave_group_by_location"/>
+                            <label for="wave_group_by_location"/>
+                        </div>
+                        <field name="wave_group_by_location" invisible="1"/>
```

Comment:
you might not need this anymore :thinking: try rebasing to the latest and check since a PR was merged a couple weeks ago to drop the need for this